link-verifier/verify-links.py

```diff
@@ -151,7 +151,11 @@ def identify_broken_links(self, files, verbose):
                 cprint(f'\t{link}','green')
 
         for link in self.external_links:
-            is_broken, status_code = test_url(link)
+            # Remove the trailing slash or trailing coma
```

Comment:
Trailing slash - So that we don't do a duplicate search of `<URL>/` and `<URL>`

Trailing coma - There are a few places I've seen links be put into files like this
`/* <COMMENT> <URL>, <MORE COMMENT> */`
Where this then breaks the URL checker, since the current regex grabs the coma